packages/augur-tools/src/flash/scripts.ts

```diff
@@ -1885,13 +1886,33 @@ export function addScripts(flash: FlashSession) {
         description: 'Name of contract',
         required: true,
       },
+      {
+        name: 'removePrefix',
```

Comment:
Why not check the first two characters and remove `0x` if present? Less prone to config issues.